src/System.Private.Xml/tests/XmlDocument/XmlAttributeCollectionTests/InsertAfterTests.cs

```diff
@@ -230,6 +231,7 @@ public void InsertAfterRemovesDupRefAttrAtTheEnd()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)] //Fix for this edge case was only made in NetCore
```

Comment:
Can we add a corresponding test that runs on NetFramework to test the old behavior?